nest-tax-backend/src/noris/subservices/noris-connection.subservice.ts

```diff
@@ -26,8 +26,11 @@ export class NorisConnectionSubservice {
     }
   }
 
-  async createConnection(): Promise<ConnectionPool> {
+  async createConnection(
+    configOverrides?: Partial<config>,
+  ): Promise<ConnectionPool> {
     const connection = await connect({
+      ...configOverrides,
```

Comment:
This will not work. JS takes the last values, and since these are first and the hardcoded 120 are last, it will always use 120 and ignore those. Move this spread to be the last, after the `options {...},` parameter.